ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -397,6 +438,24 @@ SettingsPageLayout {
                 value: view.selfDestruct
             }
 
+            Binding {
```

Comment:
IDK why I don't see this warning in storybook:
`QML Binding: Not restoring previous value because restoreMode has not been set.
This behavior is deprecated.`
I'd expect to see it 🤔 

In any case I think we should set the restoreMode.

Reply:
I cannot see the warning neither in the `app` nor in `storybook` but I will add the restore mode anyway!